fire/SFMainMod.F90

```diff
@@ -928,9 +928,10 @@ subroutine  crown_damage ( currentSite )
              
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
                 
-                ! Calculate clear branch bole height at base of crown
+                ! height_cbb = clear branch bole height at base of crown (m)
                 ! inst%crown = crown_depth_frac (PFT)
-                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
+                height_cbb  = currentCohort%hite * (1.0_r8 - EDPftvarcon_inst%crown(currentCohort%pft)
+                crown_depth = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft) 
```

Comment:
@jkshuman this looks correct to me. 
A nit-picky request would be to reorder lines 933 and 934 so as to rewrite height_cbb as:

`height_cbb = currentCohort%hite - crown_depth`

Reply:
@slevisconsulting I like it. That's what I was going for in the first place, but this makes it super clear. done.